### PR TITLE
Add argument/env var to set the session expiry for the AWS credentials.

### PR DESCRIPTION
**Pull request by tim-fitzgerald**
This change allows the user to set the expiration time for the AWS credentials retrieved by `okta-aws-cli`.

### CODE REVIEW

**Review by monde — CHANGES_REQUESTED** [review]
@tim-fitzgerald I love this PR. Making the token's ttl configurable was on my TODO list. Thanks for getting that work done.

I do think there are unintended consequences on the HEADDOC-like formatting of help output with the string literals that need to be corrected.

File: cmd/root/root.go [hunk 1]
```diff
@@ -124,10 +131,10 @@ func buildRootCommand() *cobra.Command {
 		Use:     "okta-aws-cli",
 		Short:   "okta-aws-cli - Okta federated identity for AWS CLI",
 		Long: `okta-aws-cli - Okta federated identity for AWS CLI
-
-Okta authentication for federated identity providers in support of AWS CLI.
-okta-aws-cli handles authentication to the IdP and token exchange with AWS STS 
-to collect a proper IAM role for the AWS CLI operator.`,
+
```
[the remark this answers is not in the document]
Comment (monde) [reply cmd/root/root.go:134]: `go run cmd/okta-aws-cli/main.go --help`

**Comment by tim-fitzgerald** [comment]
@monde thanks Mike! Yup overzealous VSCode formatting that I didn't notice. Have reverted those erroneous formatting changes.

**Review by monde — APPROVED** [review]
Thank you @tim-fitzgerald ! ship it

**Pull request merged**